### PR TITLE
Add wiki.facepunch.com fix

### PR DESCRIPTION
**Pull request by dankmolot**
Before fix:
https://imgur.com/QJvG365

After fix:
https://imgur.com/iNkKB1U

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -13751,6 +13751,18 @@ INVERT
 
 ================================
 
+wiki.facepunch.com
+
+CSS
+body {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:13757]: It doesn't seem `body` has any background image?

Comment (dankmolot) [reply src/config/dynamic-theme-fixes.config:13757]: wiki.facepunch.com/gmod have repeating background image (3x3) in <div class="body">. Without dark reader, this image hard to see. But when dark reader inverts this image, it is looks weird. https://imgur.com/QJvG365

Later I'm will send you devtools with css properties.

Comment (dankmolot) [reply src/config/dynamic-theme-fixes.config:13757]: \<body> doesn't have any background (color or image). But dark reader adds background color to it

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -13751,6 +13751,18 @@ INVERT
...
+    background: none !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:13758]:
```suggestion
    background-color: transparent !important;
```

**Comment by dankmolot** [comment]
I added your changes

**Review by Gusted — APPROVED** [review]

**Pull request merged**